frameioclient/py3_uploader.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import math
+import concurrent.futures
+import threading
+
+thread_local = threading.local()
+
+class FrameioUploader(object):
+  def __init__(self, asset, file):
+    self.asset = asset
+    self.file = file
+
+  def _read_chunk(self, file, size):
```

Comment:
my python is not good but I am not seeing how your determine the startByte. I only see that you define the file and the chunk size. To me it reads like you're always reading from byte 0 to size.

Reply:
This is a good question, however this section was written before my time so I don't honestly know the answer.